src/main/resources/com/nirima/jenkins/plugins/docker/builder/DockerBuilderPublisher/help-noCache.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Builds the image with --no-cache. Disables caching of layers.
```

Comment:
Please surround `--no-cache` with `<code>` ... `</code>` to make it 100% clear that we're referring to a docker argument here.
I'd suggest that you also add the words "If set" to the start, e.g. "If set, builds the image with --no-cache which disables caching of layers." so that nobody can misunderstand things.  I know this and you know this, and I expect that most users will understand this, but I'm sure that some folks won't...

A link to the https://docs.docker.com/engine/reference/commandline/build/ docs would also help, as that way folks can see what specific "no-cache" argument you're referring to here (not all users of this plugin are familiar with the docker command line).